src/boot/bootstrapper-router.tsx

```diff
@@ -4,30 +4,24 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import {
-	ModalManager,
-	ModalManagerContext,
-	SnackbarManager,
-	SnackbarManagerContext
-} from '@zextras/carbonio-design-system';
-import React, { FC, useContext, useEffect } from 'react';
+import { ModalManagerContext, SnackbarManagerContext } from '@zextras/carbonio-design-system';
+import React, { useContext, useEffect } from 'react';
 import type { TFunction } from 'i18next';
 import { useTranslation } from 'react-i18next';
-import { BrowserRouter, Route, Switch, useHistory, useParams } from 'react-router-dom';
+import { Route, Switch, useHistory, useParams } from 'react-router-dom';
 import { useBridge } from '../store/context-bridge';
 import AppLoaderMounter from './app/app-loader-mounter';
-import { BASENAME, IS_STANDALONE } from '../constants';
+import { IS_STANDALONE } from '../constants';
 import { NotificationPermissionChecker } from '../notification/NotificationPermissionChecker';
 import ShellView from '../shell/shell-view';
 import { useAppStore } from '../store/app';
 import { registerDefaultViews } from './app/default-views';
+import { Loader } from './loader';
 
-const ContextBridge: FC = () => {
+const ContextBridge = (): null => {
 	const history = useHistory();
-	// eslint-disable-next-line @typescript-eslint/ban-types
-	const createSnackbar = useContext(SnackbarManagerContext) as Function;
-	// eslint-disable-next-line @typescript-eslint/ban-types
-	const createModal = useContext(ModalManagerContext) as Function;
+	const createSnackbar = useContext(SnackbarManagerContext);
```

Comment:
can be replaced with useSnackbar
```suggestion
	const createSnackbar = useSnackbar();
```